ci-operator/step-registry/cucushift/installer/rehearse/aws/ipi/localzone/byo-subnet/day2/provision/cucushift-installer-rehearse-aws-ipi-localzone-byo-subnet-day2-provision-chain.yaml

```diff
@@ -0,0 +1,42 @@
+chain:
+  as: cucushift-installer-rehearse-aws-ipi-localzone-byo-subnet-day2-provision
+  steps:
+  - ref: ipi-install-rbac
+  - ref: openshift-cluster-bot-rbac
+  - ref: aws-provision-vpc-shared
+  - ref: ipi-conf
+  - ref: ipi-conf-telemetry
+  - ref: ipi-conf-aws-custom-vpc
+  - ref: ipi-conf-aws
+  - ref: ipi-install-monitoringpvc
+  - ref: ipi-install-install-aws
+  - ref: ipi-install-times-collection
+  - ref: aws-provision-tags-for-byo-vpc
+  - ref: enable-qe-catalogsource
+  - ref: storage-conf-storageclass-set-default-storageclass
+  - chain: edge-nodes-provision-day2-localzone-nodes
+  - ref: health-check-node-count
+  - chain: cucushift-installer-check
+  env:
+  - name: CONTROL_PLANE_INSTANCE_TYPE
+    default: "m6i.xlarge"
+    documentation: "Instance type for control plane nodes"
+  - name: COMPUTE_NODE_TYPE
+    default: "m5.xlarge"
```

Comment:
Can we let the installer choose the best instance instead of forcing it, a behavior used by users? So we'll automatically test that installer feature too.

Reply:
@mtulio Good suggestion, I'd consider to apply it into the existing Phase 2 workflow. For this workflow, as this is day2 operation, the `COMPUTE_NODE_TYPE` is used by `machineset` [manifests](https://github.com/openshift/release/blob/master/ci-operator/step-registry/ipi/conf/aws/localzone/machineset-manifests/ipi-conf-aws-localzone-machineset-manifests-commands.sh#L57) directly, installer won't be involved.